GroupProject/Main/clsMainSQL.cs

```diff
@@ -9,181 +9,167 @@ namespace GroupProject.Main
     class clsMainSQL
     {
         /// <summary>
-        /// Get all items from ItemDesc
+        /// This will Update an Invoice
         /// </summary>
         /// <returns></returns>
-        public string getAllItems()
+        public string UpdateInvoices(string TotalCost, string InvoiceNum)
         {
-            try
-            {
-                return "SELECT * from ItemDesc;";
-            }
-            catch (Exception ex)
-            {
-                throw new Exception(MethodInfo.GetCurrentMethod().DeclaringType.Name + "." + MethodInfo.GetCurrentMethod().Name + " -> " + ex.Message);
-            }
+            string sSQL = "Update Invoices SET TotalCost = " + TotalCost + " WHERE InvoiceNum =  " + InvoiceNum;
+            return sSQL;
         }
-
         /// <summary>
-        /// Get an invoice from Invoices table
+        /// This will Delete Selected Items
         /// </summary>
-        /// <param name="invoiceNum">InvoiceNum</param>
         /// <returns></returns>
-        public string getInvoice(int invoiceNum)
+        public string DeleteLineItems(string InvoiceNum)
         {
-            try
-            {
-                return "SELECT InvoiceNum, InvoiceDate, TotalCost FROM Invoices WHERE InvoiceNum = " + invoiceNum + ";";
-            }
-            catch (Exception ex)
-            {
-                throw new Exception(MethodInfo.GetCurrentMethod().DeclaringType.Name + "." + MethodInfo.GetCurrentMethod().Name + " -> " + ex.Message);
-            }
+            string sSQL = "DELETE FROM LineItems WHERE InvoiceNum = " + InvoiceNum;
+            return sSQL;
         }
-
         /// <summary>
-        /// Add invoice to Invoices table
+        /// This will Delete an Item From the Invoice through the LineItems DB
         /// </summary>
-        /// <param name="date">InvoiceDate</param>
-        /// <param name="cost">TotalCost</param>
+        /// <param name="invoiceNum"></param>
+        /// <param name="itemCode"></param>
         /// <returns></returns>
-        public string insertInvoice(DateTime date, int cost)
+        public string DeleteItemFromInvoice(string invoiceNum, string itemCode)
         {
-            try
```

Comment:
So I get a comment from him ever time about how I don't have enough try catch blocks, Do you know where exactly he wants us to add them? I agree on this, no reason to have in on formatting a string, but I just always get docked points

Reply:
He said to add them to everything. In my window, I added them to every single function I have, no matter how simple. It's kind of a pain, but really it's just a bunch of copy/ paste